pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -496,12 +496,7 @@ func (m *managerImpl) podEphemeralStorageLimitEviction(podStats statsapi.PodStat
 	}
 
 	podEphemeralStorageTotalUsage := &resource.Quantity{}
-	fsStatsSet := []fsStatsType{}
-	if *m.dedicatedImageFs {
-		fsStatsSet = []fsStatsType{fsStatsLogs, fsStatsLocalVolumeSource}
-	} else {
-		fsStatsSet = []fsStatsType{fsStatsRoot, fsStatsLogs, fsStatsLocalVolumeSource}
-	}
+	fsStatsSet := []fsStatsType{fsStatsRoot, fsStatsLogs, fsStatsLocalVolumeSource}
```

Comment:
Out of curiosity, do you know why we had this conditional in the first place?

Reply:
No I don't, and I'm a bit concerned about that.  I attempted to contact the author of this code (it appears to have been introduced about 2 years ago as part of another change, but with no explanation in the code or commit comment), but my email bounced.  Given the way this is used it's not apparent to me why there would be any reason for this kind of check in this place, although I could see it being useful elsewhere.